db/schema_tables.cc

```diff
@@ -354,6 +355,19 @@ schema_ptr scylla_tables(schema_features features) {
             sb.with_column("partitioner", utf8_type);
             offset |= 0b10;
         }
+        if (has_group0_schema_versioning) {
+            // If true, this table's latest schema was committed by group 0.
+            // In this case `version` column is non-null and will be used for `schema::version()` instead of calculating a hash.
+            //
+            // If false, this table's latest schema was committed outside group 0 (e.g. during RECOVERY mode).
+            // In this case `version` is null and `schema::version()` will be a hash.
+            //
+            // If null, this is either a system table, or the latest schema was committed
+            // before the GROUP0_SCHEMA_VERSIONING feature was enabled (either inside or outside group 0).
+            // In this case, for non-system tables, `version` is null and `schema::version()` will be a hash.
+            sb.with_column("committed_by_group0", boolean_type);
+            offset |= 0b100;
```

Comment:
Please reserve 0b100 for scylla-enterprise. It was wrong of us to pick a number from the same range, but we did.

Reply:
But, I think that after this series, we care a lot less about digest hashing. The next time we update the schema's schema, the update will be propagated via raft rather than each node calculating a new digest and hoping it will match.

Reply:
This is a local table, its schema is not propagated (by Raft or otherwise). Its schema version is calculated below: `system_keyspace::generate_schema_version(id, offset)`.

I can reserve `0b100`, although I think that any modifications to this table should always go through open source first for simplicity.